karma.conf.js

```diff
@@ -22,15 +22,15 @@ module.exports = function (config) {
     },
     customLaunchers: {
       ChromeHeadless: {
-      base: 'Chrome',
-      flags: [
-        '--headless',
-        '--disable-gpu',
-        // no-sandbox - for running under docker
-        '--no-sandbox',
-        // Without a remote debugging port, Google Chrome exits immediately.
-        '--remote-debugging-port=9222',
-        ],
```

Comment:
Do changes need to be made in karma.conf.js and package-lock.json? If this was accidentally, please exclude these files.

Reply:
It was accidentally. How could I exclude them?